.github/workflows/ci_docker.yml

```diff
@@ -13,6 +14,8 @@ on:
       - v*
 env:
   IMAGE_NAME: ${{ github.repository }}/${{ github.event.repository.name }}
+  UNIFIED_WAREHOUSE_IMAGE: docker.pkg.github.com/sanger/unified_warehouse/unified_warehouse:GPL-769-cherrypicked-samples-view-revisited
```

Comment:
We should use [services](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices) instead of pulling the images manually.

Reply:
I couldn't find the way of using a service just to run only a task. But I think I could change it to accept an ENV variable like: 
```
RESET_DATABASE=true
```
that if is present on starting the service for UWH and EWH it will reset the database and stop. What do you think?

Reply:
Changed to INTEGRATION_TEST_SETUP: true in the github workflow

Reply:
Yeah, that looks good, you can remove these now.